client/components/jetpack/card/jetpack-product-card/display-price/style.scss

```diff
@@ -128,11 +128,6 @@
 		background-color: rgba(var(--studio-jetpack-green-10-rgb), 0.2);
 	}
 
-	&__prices {
-		display: flex;
```

Comment:
I'm also not sure about the reason for introducing this styling, but we can easily maintain `nowrap` with setting `display` as `inline-flex`, instead of `flex` (that is a `block-flex`).